content_store_controller.go

```diff
@@ -75,23 +91,52 @@ func (cs *ContentStoreController) registerWithURLArbiter(path, publishingApp str
 	return true
 }
 
-func (cs *ContentStoreController) GetContentStoreRequest(w http.ResponseWriter, r *http.Request) {
-	cs.doContentStoreRequest("GET", r.URL.Path, nil, w)
-}
-
-func (cs *ContentStoreController) DeleteContentStoreRequest(w http.ResponseWriter, r *http.Request) {
-	cs.doContentStoreRequest("DELETE", r.URL.Path, nil, w)
-}
-
 // data will be nil for requests without bodies
-func (cs *ContentStoreController) doContentStoreRequest(httpMethod string, path string, data []byte, w http.ResponseWriter) {
-	resp, err := cs.contentStore.DoRequest(httpMethod, path, data)
+func (controller *ContentStoreController) doContentStoreRequest(httpMethod string, path string, data []byte, w http.ResponseWriter) {
+	resp, err := controller.liveContentStore.DoRequest(httpMethod, path, data)
+	defer resp.Body.Close()
+
 	if err != nil {
 		renderer.JSON(w, http.StatusInternalServerError, err)
 	}
```

Comment:
This wasn't introduced in the PR, but this error handling needs a `return` statement, otherwise it will attempt to send a second response after this error response.